submissions/adebayoObafemi/database/init.sql

```diff
@@ -0,0 +1,60 @@
+/* create the database for the task */
+create database foodbag;
+
+/* show the databases to confirm the creation of the intende database */
+show databases;
+
+/* select the database for use */
+USE foodbag;
+
+/* create the food_list table */
+create table food_list (
+	id int not null auto_increment,
+    food_type varchar(20) not null,
+    primary key(id)
+);
+
+/* create the vendor_food table */
+create table vendor_food (
+	id int not null auto_increment,
+    admin_id int not null,
+    food_id int not null,
+    food_name varchar(50) not null,
+    amount int not null,
+    primary key (id),
+    foreign key (food_id) references food_list (id)
+);
```

Comment:
admin_id is a foreign key in this table, but there's no reference to the admin table itself.